src/app/api/challenge/claim/route.ts

```diff
@@ -239,7 +239,10 @@ export async function POST(request: NextRequest) {
         ' function type:' +
         challenge.function_type
     );
-    return NextResponse.json({ status: 'invalid-body-params' });
+    return NextResponse.json({
```

Comment:
does this return an error object?

Reply:
hi, i updated this to return status:bool, then a message:string